cryptoinvestor/templates/assets/assets.html

```diff
@@ -5,12 +5,20 @@
 
 {% block body %}
     <main>
+        <script>
+            function get(id,asset){
+                var count = $('#'+asset.name).val();
+                window.location.href =  "/"+id+"?id="+asset.id+'&name='+asset.name+'&rate='+asset.rate+'&count='+count;
```

Comment:
Why do you need to pass both the name and id of an asset? IMO id should be enough